src/main/java/mate/controller/car/DeleteCarController.java

```diff
@@ -0,0 +1,28 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/cars/delete")
+public class DeleteCarController extends HttpServlet {
+    private static final Injector injector
+            = Injector.getInstance("mate");
+    private ManufacturerService manufacturerService
+            = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    private CarService carService
+            = (CarService) injector.getInstance(CarService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req,
+                         HttpServletResponse resp) throws ServletException, IOException {
+        carService.delete(Long.valueOf(req.getParameter("id")));
+        resp.sendRedirect(req.getContextPath() + "/cars/cars");
```

Comment:
```suggestion
        resp.sendRedirect(req.getContextPath() + "/cars");
```
I think (not sure) in Intellij you can find all occurrence of string in project and replace all so you don't need to look for them manually

Reply:
<img width="1440" alt="Screenshot 2023-01-26 at 14 49 36" src="https://user-images.githubusercontent.com/103888898/214842312-80cb8921-c0a5-4243-97bb-5ad4fbe81e6c.png">
![Screenshot 2023-01-26 at 14 49 40](https://user-images.githubusercontent.com/103888898/214842324-a790e041-339a-4404-8e08-c73f8804943d.png)
![Screenshot 2023-01-26 at 14 50 10](https://user-images.githubusercontent.com/103888898/214842334-9edecacf-dcbf-474f-bf50-519dec6fa187.png)
![Screenshot 2023-01-26 at 14 50 18](https://user-images.githubusercontent.com/103888898/214842342-efa25d89-36dc-4206-9d13-b9728cb9b378.png)
I will receive an error if I change this part